Goodreads.Tests/Clients/ShelvesClientTests.cs

```diff
@@ -10,7 +10,7 @@ public class ShelvesClientTests
 
         public ShelvesClientTests()
         {
-            ShelvesClient = Helper.GetClient().Shelves;
+            ShelvesClient = Helper.GetAuthClient().Shelves;
```

Comment:
Should change it, because new methods for ShelvesClient work only for auth clients